server/index.js

```diff
@@ -0,0 +1,452 @@
+'use strict';
+
+var util = require('util');
+
+var envvar = require('envvar');
+var express = require('express');
+var bodyParser = require('body-parser');
+var moment = require('moment');
+var plaid = require('plaid');
+
+var APP_PORT = envvar.number('APP_PORT', 8000);
+var PLAID_CLIENT_ID = envvar.string('PLAID_CLIENT_ID');
```

Comment:
are these the values that need to be changed? If so can we make it so they are more obvious
`var PLAID_CLIENT_ID = 'YOUR_CLIENT_ID';`

Reply:
Technically they can also pass in environment variables as well into the node command. Wanted to give both options